address-book.json

```diff
@@ -147,6 +150,9 @@
     },
     "ProxyFactory": {
       "address": "0x8eb543b35DE94B0E636402C7cA32947b22853eDF"
+    },
+    "SAIToken": {
+      "address": "0xaFF4481D10270F50f203E0763e2597776068CBc5"
```

Comment:
why are we missing the information we have on mainnet for rinkeby?

Reply:
I'm not inclined to think it was real information, since we aren't deploying token contracts to mainnet.